source/includes/steps-starting-compass.yaml

```diff
@@ -15,7 +15,13 @@ content: |
        * - :guilabel:`Hostname`
 
          - The hostname of the machine where the ``mongod`` instance is
-           running.
+           running. 
+
+           .. tip::
+
+              Compass automatically populates other fields in this dialog
```

Comment:
Sounds good. The reason I reversed is because this is the section where we're telling people to pop <x> into hostname field.  So, I thought it would be good to quickly get people to what values are acceptable:  The hostname, or here's a tip: enter the uri.

Also, I wanted people to know that we parse the URI to populate the other fields.